csm_test_utils/dns/dns_resolving.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import socket
+import sys
+import time
+
+import requests
+from influx_line_protocol import Metric, MetricCollection
+from ocomone.logging import setup_logger
+
+from ..common import base_parser, sub_parsers
+
+INT_DNS = "int_dns_resolving"
+
+collection = MetricCollection()
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+
+AGP = sub_parsers.add_parser("internal_dns_resolve", add_help=False, parents=[base_parser])
+AGP.add_argument("--dns_name", help="dns name of server to resolve", type=str)
+
+
+def dns_resolve(args):
+    metric = Metric(INT_DNS)
+    try:
+        socket.getaddrinfo(args.dns_name, 0, 0, 0, 0)
+    except socket.gaierror as Err:
+        metric.add_value("ips", Err)
+        metric.add_tag("dns_name", args.dns_name)
+        metric.add_tag("result", "Not Resolved")
+        collection.append(metric)
+        res = requests.post(f"{args.telegraf}/telegraf", data=str(collection), timeout=2)
```

Comment:
can we use `Client` here?
https://github.com/opentelekomcloud-infra/csm-test-utils/blob/master/csm_test_utils/common.py#L27

Reply:
for dns resolving Client is overhead

Reply:
but i using Client in host_check.py